apps/explorer/src/components/HomeMetrics/index.tsx

```diff
@@ -18,7 +18,7 @@ import { Stats, type StatsProps } from '~/ui/Stats';
 // Simple wrapper around stats to avoid text wrapping:
 function StatsWrapper(props: StatsProps) {
     return (
-        <div className="flex-shrink-0">
+        <div className="flex-shrink-0 first:pl-8">
```

Comment:
Why are you using `first` here? The first element is always just the stats component. Why not either just apply the padding directly or create a wrapping node that can hold the padding?

Reply:
Thx for the suggestion. Updated the code to have the container holding the left pad